.travis.yml

```diff
@@ -15,7 +15,6 @@ notifications:
       - gm130s@gmail.com  # Change this to yours should you copy this file.
 env:
   global:
-    - DEBUG_BASH='true' # Print bash trace
```

Comment:
For now +0 since when errors occur users now see less output. But in general I prefer brevity too, s let's apply this change in and see how users react.